### ISSUE TITLE
Flycheck, while using pylint, is reporting double errors (possible regression)

### ISSUE THREAD

**Issue by FrankBattaglia** [flycheck/flycheck#1303]
I am currently observing the same bug behavior identified in https://github.com/flycheck/flycheck/issues/495

Double-errors show up in the minibuffer, in the `*Messages*` buffer, the `*Flycheck errors*` buffer (i.e., `C-c ! l`).

```
Syntax checkers for buffer __init__.py in python-mode:

  python-flake8 (disabled)
    - may enable:         Automatically disabled!
    - executable:         Not found
    - configuration file: Not found

  python-pylint
    - may enable:         yes
    - executable:         Found at /usr/local/bin/pylint
    - configuration file: Not found

  python-pycompile
    - may enable: yes
    - executable: Found at /usr/local/bin/python

Flycheck Mode is enabled.  Use C-u C-c ! x to enable disabled checkers.

--------------------

Flycheck version: 31snapshot (package: 20170817.57)
Emacs version:    25.1.1
System:           x86_64-apple-darwin15.5.0
Window system:    ns
```
**Operating System**: OSX 10.12 (Sierra)
**Emacs version**: `GNU Emacs 25.1.1 (x86_64-apple-darwin15.5.0, NS appkit-1404.47 Version 10.11.5 (Build 15F34)) of 2016-09-19`
**Flycheck version**: `Flycheck version: 31snapshot (package: 20170817.57)`
```
$ pylint --version
No config file found, using default configuration
pylint 1.7.2, 
astroid 1.5.3
Python 2.7.12 (default, Jul 25 2016, 17:16:23) 
[GCC 4.2.1 Compatible Apple LLVM 7.3.0 (clang-703.0.31)]
```

**Comment by FrankBattaglia** [comment]
Upon further testing it appears that this behavior is limited to `info` messages (`warning` and `error` messages do not appear duplicated)

**Comment by cpitclaudel** [comment]
Hi! Thanks for reporting; sorry for the trouble. Could you come up with a sample Python file that I could reproduce this bug on? Bonus points if it only requires Flycheck (and no other Emacs packages) :)

**Comment by FrankBattaglia** [comment]
Coming back to this after a bit and now I can't reproduce the problem.  Not sure what to make of it, but I guess closed for now?

**Comment by cpitclaudel** [comment]
Indeed. Please reopen if it happens again!
